pyproject.toml

```diff
@@ -108,5 +103,10 @@ module = [
 ]
 ignore_errors = true
 
+[tool.poetry.plugins."console_scripts"]
```

Comment:
is there a difference between 

tool.poetry.plugins."console_scripts"
and
tool.poetry.scripts

Reply:
I don't know, I don't know the outcome is the same (from a users perspective).
I do not recall why "console_scripts" was chosen, anyway I needed to remove it in one of the sections because of the double definition. Because I personally feel `tool.poetry.script` is better I have chosen to put it there.